packages/api-server/src/plugins/api.ts

```diff
@@ -54,6 +56,11 @@ export async function redwoodFastifyAPI(
     }
   }
 
+  // Run users custom server configuration function
```

Comment:
Note @Josh-Walker-GM and I found that the order where custom server config was added matters - if after loading functions, then plugins like fastify complress did not work properly.